backend/src/main/java/me/quizzl/backend/models/Answer.java

```diff
@@ -0,0 +1,47 @@
+package me.quizzl.backend.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+@Entity
+public class Answer {
+   
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name="id", updatable = false, nullable = false)
+    protected Long id;
+
+    @Column(name = "question_string")
+    protected String questionStr;
```

Comment:
instead of using the question string to identify the question which this answer is answering, should we use the id of the question instead and make it like a many to one relationship (one question has many "answers")